go/agent/agent_dao.go

```diff
@@ -164,7 +164,7 @@ func ReadOutdatedAgentsHosts() ([]string, error) {
 		from
 			host_agent
 		where
-			IFNULL(last_checked < now() - interval ? minute, true)
+			IFNULL(last_checked < now() - interval ? minute, 1)
```

Comment:
this change need for SQLlite compatibility cause SQLite desn't have BOOLEAN literal